panther_analysis_tool/main.py

```diff
@@ -259,6 +259,84 @@ def upload_analysis(args: argparse.Namespace) -> Tuple[int, str]:
     return 0, ''
 
 
+def update_schemas(args: argparse.Namespace) -> Tuple[int, str]:
+    """Updates managed schemas in a Panther deployment.
+
+    Returns 1 if the update fails.
+
+    Args:
+        args: The populated Argparse namespace with parsed command-line arguments.
+
+    Returns:
+        A tuple of return code and the archive filename.
+    """
+
+    # optionally set env variable for profile passed as argument
+    # this must be called prior to setting up the client
+    if args.aws_profile is not None:
+        logging.info('Using AWS profile: %s', args.aws_profile)
+        set_env("AWS_PROFILE", args.aws_profile)
+
+    client = boto3.client('lambda')
+    logging.info('Fetching updates')
+    response = client.invoke(FunctionName='panther-logtypes-api',
+                             InvocationType='RequestResponse',
+                             Payload=json.dumps(
+                                 {'ListManagedSchemaUpdates': {}}))
+    response_str = response['Payload'].read().decode('utf-8')
+    response_payload = json.loads(response_str)
+
+    api_err = response_payload.get('error')
+    if api_err is not None:
+        logging.error(
+            'Failed to list managed schema updates\n\tcode: %s\n\terror message: %s',
+            api_err['code'], api_err['message'])
+        return 1, ''
+
+    releases = response_payload.get('releases')
+    if not releases:
+        logging.info('No updates available.')
+        return 0, ''
+
+    tags = [r.get('tag') for r in releases]
+    latest_tag = tags[-1]
+    while True:
+        print('Available versions:')
+        for tag in tags:
+            print('\t%s' % tag)
+            print(
+                'Panther will update managed schemas to the latest version (%s)'
+                % tag)
```

Comment:
Should this be unindented one tab? It seems this will print this line after every single available version.

Reply:
True. I think it was like this initially, probably a code reformat changed it ...

Reply:
Fixed, this originally correct but I mistakenly nudged it in during `make fmt` issues